client/securedrop_client/config.py

```diff
@@ -39,11 +39,13 @@ class Config:
         "gpg_domain": "QUBES_GPG_DOMAIN",
         "journalist_key_fingerprint": "SD_SUBMISSION_KEY_FPR",
         "download_retry_limit": "SD_DOWNLOAD_RETRY_LIMIT",
+        "proxy_vm_name": "SD_PROXY_VM_NAME",
```

Comment:
Very nit:  Now that we have two keys that refer to other VMs, do we want to standardize their naming?  [`QUBES_GPG_DOMAIN` isn't up to us.](https://www.qubes-os.org/doc/split-gpg/#configuring-the-client-apps-to-use-split-gpg-backend)  Is `SD_*_VM_NAME` the convention we want to use for the ones that are?  I would probably accept the asymmetry (as we do for `SD_SUBMISSION_KEY_FPR`) of:

```suggestion
        "proxy_vm_name": "SD_PROXY_VM",  # or "SD_PROXY_DOMAIN"
```